### PR TITLE
Add lizardfolk as a NPC race

### PR DESCRIPTION
**Pull request by sytone**
## What does this do?

Add Lizardfolk as a NPC race

## How was this tested? Did you test the changes in the compiled `.html` file?

Built and tested locally by generating NPCs and re-generating towns to validate population percentages.

## Is there a [GitHub Issue](https://github.com/ryceg/Eigengrau-s-Essential-Establishment-Generator/issues?q=is%3Aopen+is%3Aissue) that this is resolving?

No

### CODE REVIEW

**Review by ryceg — APPROVED** [review]

File: src/NPCGeneration/createNPC.ts [hunk 1]
```diff
@@ -57,7 +57,10 @@ export const createNPC = (town: Town, base = defaultBase): NPC => {
   const profession = base.profession || lib.fetchProfessionChance(town, base as NPC)
 
   const firstName = base.firstName || getFirstName(race, base.gender)
-  const lastName = base.lastName || getLastName(race)
+  let lastName = base.lastName || getLastName(race)
+  if (lastName === firstName) {
+    lastName = firstName
```
Comment (Glinkis) [inline src/NPCGeneration/createNPC.ts:62]: This assignment does nothing. Why is this here?

Comment (sytone) [reply src/NPCGeneration/createNPC.ts:62]: It should be is lastName === '' then lastname = firstname.

Comment (ryceg) [reply src/NPCGeneration/createNPC.ts:62]: Just edited your comment to make it clearer :)
